restapi/src/main/java/io/stargate/web/models/Filter.java

```diff
@@ -40,6 +42,7 @@ public void setColumnName(String columnName) {
     this.columnName = columnName;
   }
 
+  @ApiModelProperty(value = "")
```

Comment:
The operator enum values don't make it into the swagger docs. Should we show and describe the different values here?